edgelet/hsm-sys/azure-iot-hsm-c/src/constants.c

```diff
@@ -26,4 +29,6 @@ const char* const OWNER_CA_COMMON_NAME = "Test Edge Owner CA";
 const char* const DEVICE_CA_COMMON_NAME = "Test Edge Device CA";
 const char* const EDGE_CA_ALIAS = "iotedged_ca";
 const char* const EDGE_CA_COMMON_NAME = "Test Iotedge CA";
+const char* const EDGE_DEVICE_ALIAS = "edgelet_device";
```

Comment:
What is this constant used for?

Reply:
We use this alias to associate/label the device cert. This is used when generating the device identity certificate using the provided CA certificates.